python/social_bot/envs/simple_navigation.py

```diff
@@ -157,9 +157,9 @@ def step(self, action):
             sentence = ''
             controls = action
         controls = dict(zip(self._joint_names, controls))
-        teacher_action = self._teacher.teach(sentence)
```

Comment:
I'm not very sure here. It seems "teacher.teach()" should be after the call of "world.step()". As the teacher_action.reward should be the calculated from the observation after action being taken.